pkg/helm/release/manager.go

```diff
@@ -46,10 +46,10 @@ type Manager interface {
 	IsInstalled() bool
 	IsUpdateRequired() bool
 	Sync(context.Context) error
-	InstallRelease(context.Context) (*rpb.Release, error)
-	UpdateRelease(context.Context) (*rpb.Release, *rpb.Release, error)
+	InstallRelease(context.Context, ...InstallOption) (*rpb.Release, error)
+	UpdateRelease(context.Context, ...UpdateOption) (*rpb.Release, *rpb.Release, error)
 	ReconcileRelease(context.Context) (*rpb.Release, error)
-	UninstallRelease(context.Context) (*rpb.Release, error)
+	UninstallRelease(context.Context, ...UninstallOption) (*rpb.Release, error)
```

Comment:
If I understood the @joelanford's suggestion, all here would receive something such as ReleaseOptions, am I right @joelanford?

Reply:
Uninstall, Install and Update have different options